src/content/docs/en/developers/guides/running-a-scroll-node.mdx

```diff
@@ -117,7 +118,8 @@ Running the node in Docker might have a significant impact on node performance.
         --datadir "/volume/l2geth-datadir" \
         --gcmode archive --cache.noprefetch \
         --http --http.addr "0.0.0.0" --http.port 8545 --http.api "eth,net,web3,debug,scroll" \
-        --l1.endpoint "$L2GETH_L1_ENDPOINT" --rollup.verify
+        --l1.endpoint "$L2GETH_L1_ENDPOINT" --rollup.verify \
+        --gpo.congestionthreshold=500
```

Comment:
same here

Reply:
cc @yiweichi -- my guess is we should change this in our SDK RPC values as well?